src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -192,8 +193,8 @@ If after those retries the listener is still failing to perform its operation, t
 [Event Dead Letter](manage-webadmin.html#Event_Dead_Letter). 
 This API allows diagnosing issues, as well as redelivering the events. 
 
-To check that you have undelivered events in your system, you can first
-[list mailbox listener groups](manage-webadmin.html#Listing_mailbox_listener_groups).
+To check that you have undelivered events in your system, you can first check with [dead_letter event_health_check](manage-webadmin.html#Event_Dead_Letter)
+then [list mailbox listener groups](manage-webadmin.html#Listing_mailbox_listener_groups).
```

Comment:
You can explore Event DeadLetter content through WebAdmin. For this, [list mailbox listener groups](...)

You will...